### PR TITLE
Download bash script

### PR DESCRIPTION
**Pull request by Volodimirich**

### CODE REVIEW

File: test/scripts/download.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    fi
+
+    if ! [ -f "./dblp.v13.7z" ]
+    then
+        wget https://originalstatic.aminer.cn/misc/dblp.v13.7z
+    fi
+fi
+
+apt-get update
+apt-get install apt-get install p7zip-full
```
Comment (besteady) [inline test/scripts/download.sh]:
```suggestion
apt-get install p7zip-full
```

File: test/scripts/download.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+7z x dblp.v13.7z -o../../dataset/
```
Comment (besteady) [inline test/scripts/download.sh]:
```suggestion
7z x dblp.v13.7z -o ../../dataset/
```

File: test/scripts/download.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+rm dblp.v13.7z
```
Comment (besteady) [inline test/scripts/download.sh]: probably save this file?

Comment (Volodimirich) [reply test/scripts/download.sh]: And check, if we have it in our directory, we need only unzip and sed. Seems logic

Comment (Volodimirich) [reply test/scripts/download.sh]: But still, this archive is very big

**Pull request closed without merging**